<sdf version="1.4">
    <model name="[marker_name]">
        <pose>0 0 0 0 0 0</pose>
        <static>true</static>
        <link name="base">
            <inertial>
                <mass>1.0</mass>
                <inertia>
                    <ixx>0.1</ixx> <ixy>0.0</ixy> <ixz>0.0</ixz>
                    <iyy>0.1</iyy> <iyz>0.0</iyz>
                    <izz>0.1</izz>
                </inertia>
            </inertial>
            <collision name="collision">
                <geometry>
                    <box>
                        <size>[side_length] [side_length] 0.01</size>
                    </box>
                </geometry>
            </collision>
            <visual name="visual">
                <geometry>
                    <mesh>
                        <uri>model://[marker_name]/meshes/unit_plane.dae</uri>
                        <scale>1 [side_length] [side_length]</scale>
                    </mesh>
                </geometry>
            </visual>
        </link>
    </model>
</sdf>